src/azul/indexer/notify_service.py

```diff
@@ -0,0 +1,46 @@
+import json
+import logging
+
+from azul import (
+    JSON,
+    config,
+)
+from azul.deployment import (
+    aws,
+)
+from azul.strings import (
+    trunc_ellipses,
+)
+
+log = logging.getLogger(__name__)
+
+
+class AzulEmailNotificationService:
+
+    def notify_group(self, subject: str, message: str) -> None:
+        log.info('Notifying group of event %r', trunc_ellipses(message, 256))
+        # `message` is deserialized to a Python object to be re-serialized as a
+        # printable JSON
+        body = json.loads(message)
+        response = aws.ses.send_email(
+            FromEmailAddress=f'notify@{config.domain_name}',
+            Destination={
+                'ToAddresses': [config.monitoring_email]
+            },
+            Content=self._format_content(subject, json.dumps(body, indent=4))
+        )
+        log.info('Notification sent, %r', response['MessageId'])
+
+    def _format_content(self, subject: str, body: str) -> JSON:
```

Comment:
This method doesn't seem to do any formatting.
```suggestion
    def _content(self, subject: str, body: str) -> JSON:
```